src/Command/User/AddUserCommand.php

```diff
@@ -149,6 +156,39 @@ protected function interact(InputInterface $input, OutputInterface $output): voi
             $fullName = $this->io->ask('Full Name', null, [$this->validator, 'validateFullName']);
             $input->setArgument('full-name', $fullName);
         }
+
+        $helper = $this->getHelper('question');
+
+        // Ask for the role if it's not defined
+        $role = $input->getArgument('role');
+        if (null !== $role) {
+            $this->io->text(' > <info>Role</info>: '.$role);
+        } else {
+            $question = new ChoiceQuestion(
+                'Please select the users role (defaults to editor)',
+                ['editor', 'admin'],
+                0
+            );
+            $question->setErrorMessage('Color %s is invalid.');
+
+            $role = $helper->ask($input, $output, $question);
+            $output->writeln('You have just selected: '.$role);
+        }
+
+        // Ask for the tenant keys if it's not defined
+        $tenantKeys = $input->getArgument('tenant-keys');
+        if (0 < \count($tenantKeys)) {
+            $this->io->text(' > <info>Tenant Keys</info>: '.$tenantKeys);
+        } else {
+            $question = new ChoiceQuestion(
+                'Please select the tenant(s) the user should belong to (to select multiple answer with a list. E.g: "key1, key3")',
```

Comment:
I don't understand what “(to select multiple answer with a list. E.g: "key1, key3")” means. Doesn't the helper help the user picking multiple values?

Reply:
No. You have to answer with a list of comma separated values. There is no "UI".

Reply:
https://symfony.com/doc/current/components/console/helpers/questionhelper.html#multiple-choices

```
use Symfony\Component\Console\Question\ChoiceQuestion;

// ...
public function execute(InputInterface $input, OutputInterface $output): int
{
    // ...
    $helper = $this->getHelper('question');
    $question = new ChoiceQuestion(
        'Please select your favorite colors (defaults to red and blue)',
        ['red', 'blue', 'yellow'],
        '0,1'
    );
    $question->setMultiselect(true);

    $colors = $helper->ask($input, $output, $question);
    $output->writeln('You have just selected: ' . implode(', ', $colors));

    return Command::SUCCESS;
}
```

Reply:
Notice `implode(', ', $colors)`

Reply:
But yes, before testing it I also expected a checkbox like feature